geoh5py/groups/property_group_type.py

```diff
@@ -132,26 +133,48 @@ def verify(cls, children: list[Data]):
             )
 
 
-# mapper for group types
 GROUP_TYPES = {
```

Comment:
I have the feeling there is some repetition here betwen GROUP_TYPE, the objects of the Enum and the name

Reply:
Right, we can probably remove the name from the classes....